### ISSUE TITLE
Add GUI to activate/deactivate of default rules for apps

### ISSUE THREAD

**Issue by warpech** [Starcounter/Blending#241]
The GUI will present the default rules in form of a table. The only editable part is a checkbox that turns the rule to be active or not. 

The table should distinguish from the initial state and the current state. The initial state is the one that is registered through `blend.json` or `Blender` API. 

As part of this task, a new data model class needs to be created. This class will hold the data needed to identify default rules registered through `blend.json` or `Blender` API.

This configuration should be part of import/export.

This work should only be done in the `mapping` branch.

UI will look somehow like:

```
Uri                            | AppName    | Contexts             | Dir   | Active
/people/partial/settings/{?}   | People     | fullpage, settings   | -->   | [ ]   
```

**Comment by kegor** [comment]
According to this https://github.com/Starcounter/Blending/pull/250#issuecomment-404127350 additional changes will be tracked in a new PR.

**Comment by kegor** [comment]
Closing because additional changes were merged in the #259.
